### PR TITLE
add event

### PR DESCRIPTION
**Pull request by gpsanant**
we need to emit this for the backend to work

### CODE REVIEW

File: src/contracts/pods/EigenPod.sol [hunk 1]
```diff
@@ -164,6 +164,7 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
          * initialized with hasRestaked set to true.
          */
         hasRestaked = true;
+        emit RestakingActivated(podOwner);
```
Comment (ChaoticWalrus) [inline src/contracts/pods/EigenPod.sol:167]: IIRC we set `hasRestaked = true` in other places as well!

Comment (ChaoticWalrus) [reply src/contracts/pods/EigenPod.sol:167]: are those just already covered then?

Comment (gpsanant) [reply src/contracts/pods/EigenPod.sol:167]: ye we set it at one other place, and we are only emitting there before this change :)

**Comment by ChaoticWalrus** [comment]
LGTM

**Review by ChaoticWalrus — APPROVED** [review]
LGTM'd

**Pull request merged**